hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -55,6 +55,7 @@
   private static final String DEFAULT_PARALLELISM = "1500";
   private static final String INSERT_PARALLELISM = "hoodie.insert.shuffle.parallelism";
   private static final String BULKINSERT_PARALLELISM = "hoodie.bulkinsert.shuffle.parallelism";
+  private static final String BULKINSERT_USER_DEFINED_PARTITIONER_CLASS = "hoodie.bulkinsert.user_defined.partitioner.class";
```

Comment:
rename to `.user.defined.partitioner.class`

Reply:
Sure, I will change it.

Reply:
it has been updated.